Issue#10: No configparser in Python2

chenl opened the pull request:
configparser can't be imported in Python2, so a fallback in required.

chenl committed:
Issue#10: No configparser in Python2

configparser can't be imported in Python2, so a fallback in required.

sdague reviewed (changes requested):
Instead of doing things this way, can you use six.moves.configparser ? - https://pythonhosted.org/six/#module-six.moves 

That will do this without the exception handling.

chenl commented:
I didn't know if you have it already as a dependency, and I didn't want to introduce another one.

chenl committed:
Fixed for Issue#10: No configparser in Python2 #11

Compliance to comments by @sdague with regard to PR#11: use `six.moves`
instead of `try: ...; except ImportError`.

Also declare dependencies.

sdague commented:
thanks!

sdague commented:
Released in 0.5.0 on pypi now

chenl commented:
Grate,

Verified that it is working, and added it in my requirements.txt :)

Thanks.

On Thu, Oct 20, 2016 at 6:01 PM, Sean Dague notifications@github.com
wrote:

> Released in 0.5.0 on pypi now
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/sdague/amt/pull/11#issuecomment-255131999, or mute
> the thread
> https://github.com/notifications/unsubscribe-auth/ABmZU6BTb4M30N4zScnfLlxBrfAxvC3Hks5q14JTgaJpZM4KbHPm
> .

## 

Cheers,
Chen.
